crates/store/re_grpc_client/src/lib.rs

```diff
@@ -376,3 +387,104 @@ async fn stream_catalog_async(
 
     Ok(())
 }
+
+// Craft a blueprint from relevant chunks and activate it
+fn activate_catalog_blueprint(
+    tx: &re_smart_channel::Sender<LogMsg>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    let blueprint_store_id =
+        StoreId::from_string(StoreKind::Blueprint, "catalog_blueprint".to_owned());
+    let blueprint_store_info = StoreInfo {
+        application_id: ApplicationId::from("redap_catalog"),
+        store_id: blueprint_store_id.clone(),
+        cloned_from: None,
+        is_official_example: false,
+        started: Time::now(),
+        store_source: StoreSource::Unknown,
+        store_version: None,
+    };
+
+    if tx
+        .send(LogMsg::SetStoreInfo(SetStoreInfo {
+            row_id: *re_chunk::RowId::new(),
+            info: blueprint_store_info,
+        }))
+        .is_err()
+    {
+        re_log::debug!("Receiver disconnected");
+        return Ok(());
+    }
+
+    let timepoint = [(Timeline::new_sequence("blueprint"), 1)];
+
+    let vb = ViewBlueprint::new("Dataframe")
+        .with_visible(true)
+        .with_space_origin("/");
+
+    let view_uuid = uuid::Uuid::new_v4();
+    let view_id = format!("/view/{view_uuid}");
```

Comment:
These entities are typically are built using `SpaceViewId::registry_path()`, where SpaceViewId is a wrapper over these uuids.

Reply:
also, `view_id` is misleading. should be `view_entity_path`